Cargo.toml

```diff
@@ -7,7 +7,7 @@
 members = [
   # core
   "core/protobuf_data_access",
-  "core/in-vehicle-digital-twin",
+  "core/invehicle-digital-twin",
```

Comment:
I think it makes more sense if `in-vehicle` is hyphenated.

Reply:
When I use underscores rather than hyphens (e.g. invehicle_digital_twin) I do not use an underscore between "in" and "vehicle" because it seems to be excessive.  I chose to keep the underscore and hyphen forms consistent with each other.